cmd/convox/rack.go

```diff
@@ -215,7 +221,19 @@ func cmdRackParamsSet(c *cli.Context) error {
 		return stdcli.Error(err)
 	}
 
-	fmt.Println("OK")
+	if c.Bool("wait") {
+		stdcli.Startf("Waiting for completion")
+
+		// give the rack a few seconds to start updating
+		time.Sleep(5 * time.Second)
```

Comment:
I just used the same sleep time as [here](https://github.com/convox/rack/blob/master/cmd/convox/rack.go#L304-L315). I haven't tested with 2-3 seconds, but can if you think 5 seconds is too long here.